### DISCUSSION TITLE
Add Seat by API

### DISCUSSION THREAD

**Discussion by gogelDE** [pretix/pretix#3495] [Q&A]
Hello, I am trying to assign a seat to my positions via the API.
I am using the endpoint specified in the documentation:
PATCH /api/v1/organizers/(organizer)/events/(event)/orderpositions/(id)/

How should the body of my request look like for the API to accept it? Currently, I'm getting the following error:
"seat": [
        "Not a valid string."
    ]

My request body:

"seat" : {
        "name" : "A1",
        "seat_guid" : "Groundfloor-A-1"
    }

I appreciate your response.

**Comment by raphaelm** [comment]
You can just pass

```
"seat": "Groundfloor-A-1"
```
